internal/linker/test/double_workspace_link/test.js

```diff
@@ -0,0 +1,13 @@
+const npm = require("npm");
```

Comment:
The module itself does not really matter, its any module that is in `node_modules`. I picked `npm` at random from the set of deps in package.json in this repo